index.js

```diff
@@ -1,4 +1,4 @@
-var toString = {}.toString;
+var toString = Object.prototype.toString;
```

Comment:
this change is now correct, but it's actually *less* robust, I think, since it relies on `Object` being unmodified as *well* as `Object.prototype`. `{}.toString` only relies on the original `Object.prototype` being unmodified.